.github/workflows/generators.yml

```diff
@@ -23,3 +23,18 @@ jobs:
       - name: Install project
         run: |
           poetry install --no-interaction
+      - name: Get pinecards
+        run: |
+          cd examples
+          # https://github.com/actions/checkout/issues/24
+          git config --global url.https://github.com/.insteadOf git://github.com/
+          git clone https://github.com/NNPDF/pinecards.git
```

Comment:
Use [`actions/checkout`](https://github.com/actions/checkout), it's the official one :)

It does support a `repository` key, that allows to choose a repo different from the current one.

Reply:
Ah sorry, you already put the reference, I haven't seen the comment at first.

But the issue is solved, it should be fine, isn't it?

Reply:
```suggestion
      - name: Fetch Pinecards
        use: actions/checkout@v3
        with:
          repository: NNPDF/pinecards
          path: examples/
```